OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -372,6 +372,7 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
   if (value)
     {
       NSArray *recipients;
+      int ccFirstIndex;
```

Comment:
If a int is being used in the for, and this is only used to compare against that other value, I really don't see any value in using NSUInteger here...